src/main/java/labs/introtoprogramming/lab5/scene/PointLight.java

```diff
@@ -1,4 +1,19 @@
 package labs.introtoprogramming.lab5.scene;
 
-public abstract class PointLight extends SceneObject {
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+public class PointLight extends Light {
+  public PointLight(Transform transform, double intensity) {
+    super(transform, intensity);
+  }
+
+  @Override
+  public double illuminate(Vector3 normal, double distance) {
+    return intensity() / (4 * Math.PI * Math.pow(distance, 2));
```

Comment:
This value need to be multiply by cos or in other words by Math.max(intensity() * normal.dotProduct(getTransform().rotation().multiply(-1)), 0) as for DistantLigth.